### PR TITLE
Feature/login

### PR DESCRIPTION
**Pull request by ALerumo**

### CODE REVIEW

File: chatterbox/src/App.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ class App extends Component {
     this.funcVar=setInterval(this.splash,2000);
   }
   splash(){
-  	 history.push("/auth/login");  	 
+  	 //history.push("/auth/login");
```
Comment (jaderabbit) [inline chatterbox/src/App.js]: don't leave unnecessary comments

File: chatterbox/src/Auth/Login/index.js [hunk 2]
```diff
@@ -1,12 +1,64 @@
...
+  handleChange = panel => (event, expanded) => {
+    this.setState({
+      expanded: expanded ? panel : false,
+    });
+  };
+  register(){
+    this.props.history.push("/auth/register")
+  }
   render() {
+      const { expanded } = this.state;
```
Comment (jaderabbit) [inline chatterbox/src/Auth/Login/index.js]: indentation

File: chatterbox/src/Auth/Login/index.js [hunk 3]
```diff
@@ -1,12 +1,64 @@
...
+import { withStyles } from 'material-ui/styles';
+import MenuItem from 'material-ui/Menu/MenuItem';
+import TextField from 'material-ui/TextField';
+import Button from 'material-ui/Button';
+
+
+
 
 class Login extends Component {
+  state = {
```
Comment (jaderabbit) [inline chatterbox/src/Auth/Login/index.js]: good class-wise components

File: chatterbox/src/Auth/Login/login.css [hunk 4]
```diff
@@ -14,3 +14,37 @@
...
+.Login button {
+  margin-top: 60px;
+  width: 190px;font-size: 30px;
+  height: 50px;
+  background-color: #FB6902;
+  color: white;
+  font-weight: 500;
+  font-size: 20px;
+}
+.bottom-text {
```
Comment (jaderabbit) [inline chatterbox/src/Auth/Login/login.css]: guys, indentation, seriously

File: chatterbox/src/Main/MainArea/ChattingComponent/ChatArea/MessageLine.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+	constructor(e){
+		super(e);
+		let chr=this.props.type.toLowerCase().charAt(0);
+		let type;
+		if (chr=='s') {
+			type="sent";
+		} else if(chr=='r'){
+			type="received";
+		}
+		console.log("TYPE:",type)
```
Comment (jaderabbit) [inline chatterbox/src/Main/MainArea/ChattingComponent/ChatArea/MessageLine.js]: remove stray console.logs

File: chatterbox/src/Main/MainArea/ChattingComponent/ChatArea/ChatArea.css [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+.ChatArea .writer-container{
+	width: 100%;
+	height: 230px;
+	position: absolute;
+	bottom: 0;
+}
+
+.ChatArea .writer-container .add-attachment{
```
Comment (jaderabbit) [inline chatterbox/src/Main/MainArea/ChattingComponent/ChatArea/ChatArea.css]: can you group these in a hierarchical structure?  they get a bit deep. not serious though, just nit-picking
```
.ChatArea {
     .writer-container {

     }
}
```

**Review by jaderabbit — CHANGES_REQUESTED** [review]

File: chatterbox/src/Main/MainArea/ChattingComponent/ChatArea/ChatArea.css (hunk 6 above)
Comment (MahlekenyaneTseole) [reply chatterbox/src/Main/MainArea/ChattingComponent/ChatArea/ChatArea.css]: Is that not scss syntax? does it work for normal css?

File: chatterbox/src/App.js [hunk 7]
```diff
@@ -20,8 +20,7 @@ class App extends Component {
     console.log("TO-DO: Check if user online");
     this.funcVar=setInterval(this.splash,2000);
   }
-  splash(){
-  	 history.push("/auth/login");  	 
+  splash(){	 
   	 this.setState({loading:false})
```
Comment (jaderabbit) [inline chatterbox/src/App.js]: indentation

**Review by jaderabbit — CHANGES_REQUESTED** [review]
Indentation through out. I'm not here to baby and chase you you for this. Install a linter pls

File: chatterbox/src/App.js (hunk 7 above)
Comment (KimberleighBeyl) [reply chatterbox/src/App.js]: No SETSTATE! Use redux

File: chatterbox/src/Auth/Login/login.css [hunk 8]
```diff
@@ -12,5 +12,35 @@
...
+  width: 190px;font-size: 30px;
+  height: 50px;
+  background-color: #FB6902;
+  color: white;
+  font-weight: 500;
+  font-size: 20px;
+}
 
+.bottom-text {
+	color: #ffffff;
```
Comment (jaderabbit) [inline chatterbox/src/Auth/Login/login.css]: indentation still differs here. if you fix this you can merge

File: chatterbox/src/Auth/Register/steptwo.js [hunk 9]
```diff
@@ -1,50 +1,67 @@
...
+        let self = this;
+        //alert(ext)
+        if (
+            input.files &&
+            input.files[0] &&
+            (ext == 'gif' || ext == 'png' || ext == 'jpeg' || ext == 'jpg')
+        ) {
+            var reader = new FileReader();
+            reader.onload = function(e) {
+                // $('.imagepreview').attr('src', e.target.result);
```
Comment (jaderabbit) [inline chatterbox/src/Auth/Register/steptwo.js]: remove this

**Review by jaderabbit — APPROVED** [review]
Make the little changes I've asked for then you can merge!
Looking good! :+1:

File: chatterbox/src/Auth/Register/index.js [hunk 10]
```diff
@@ -1,84 +1,93 @@
...
+                }
+            ]
+        };
+        this.handleNext = this.handleNext.bind(this);
+    }
+    handleCanNext(canNext) {
+        this.setState({ canNext, error: false });
+    }
+    handleNext(curr) {
+        // alert("stage "+curr)
```
Comment (jaderabbit) [inline chatterbox/src/Auth/Register/index.js:39]: remove this

**Review by jaderabbit — APPROVED** [review]

**Pull request merged**